### DISCUSSION TITLE
How to copy errors?

### DISCUSSION THREAD

**Discussion by mihalt** [reloadware/reloadium#109] [Q&A]
Hi! Say me, please, how can I copy error, that I got? 
![изображение](https://user-images.githubusercontent.com/30621622/219872359-c78b0a77-9970-4826-a96d-dd1cd090501e.png)

**Comment by dkrystki** [comment]
Currently there is no way to copy that text from the editor.
An easy workaround is to copy the error message from the stdout from the output.

**Reply by mihalt** [reply]
so, I can't understand from what place is the error too, right?

**Comment by dkrystki** [comment]
Pycharm console
![2023-03-18_10:46:57](https://user-images.githubusercontent.com/45499236/226074230-3cb0d967-301f-497f-b633-ea30ec8466c5.png)
